src/gui_main.py

```diff
@@ -0,0 +1,311 @@
+# native imports
+import atexit
+import os
+import sys
+import tkinter
+from tkinter import filedialog
+
+# third party imports
+import cript
+import eel
+import requests
+
+# my imports
+from python.excel_uploader_main import ExcelUploader
```

Comment:
Minor detail: there's no need to add comments describing the types of imports, since this grouping is described in [PEP 8](https://peps.python.org/pep-0008/#imports).